bin/walli-cli

```diff
@@ -17,6 +17,11 @@ require('yargs')
       default: 9000,
       type: 'number',
     },
+    tlscertpath: {
```

Comment:
We should either use `camelCase` or just lower case in `wallid` and `walli-cli` and not mix between the two of them. What about migrating every option to all lower case in this PR @ImmanuelSegol ?

Reply:
I agree lower case is better in my opinion.